coveo-stew/coveo_stew/environment.py

```diff
@@ -50,31 +50,33 @@ def __init__(self, environment_path: Union[Path, str]) -> None:
         self.python_executable: Path = python_path
 
     @lru_cache()
-    def _guess_path(self, tool: PythonTool) -> Path:
+    def guess_path(self, tool: Union[PythonTool, str]) -> Path:

```

Comment:
Ẁhy did you switch this function from private to public? :thinking:

Reply:
oh, I used that originally, but then changed the angle. I'll revert it.